Feature/GitHub commit statuses

lowsky opened the pull request:
Hi Clay,
thanks for your work. 
I needed further infos from github, so I extended it for fetching Statuses of a Github-Commit.

I additionally extended the built-in playground-query for GitHub to contain all latest features :-)

Have fun!

Commit:
Add a retrieval of commit statuses

Commit:
adapt demo playground query for github

clayallsopp commented on graphqlhub-schemas/src/github.js:
```diff
@@ -118,6 +131,26 @@ let CommitType = new GraphQLObjectType({
+                      const {
+                          id, url, state, context, description, target_url, updated_at
+                        } = status;
+                      return {
```
if we're not actually mapping the status yet, is there a reason to not just `return statuses`?

lowsky replied:
Sure I just wanted to be careful :-) while I had some strange issues when using it with relay (#35)
I'll adapt it.

Commit:
simplify fetching and resolving statuses for repo/branch

Commit:
add unit test for github.

this also runs tests against a real github repo, so authentication needs to be setup in the .env file.

clayallsopp commented:
@lowsky thanks for the updates!
